src/components/dialogs/network-modifications/line-split-with-voltage-level/line-split-with-voltage-level-illustration.tsx

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useIntl } from 'react-intl';
+import { AccordionIllustration } from '../widget/AccordionIllustration';
+import React, { useEffect, useState } from 'react';
+import { ReactComponent as LineSplitDiagram } from '../../../../images/network-modifications/illustrations/line-split-with-voltage-level.svg';
+import { useSelector } from 'react-redux';
+import { ReduxState } from '../../../../redux/reducer.type';
+import { PARAM_THEME } from '../../../../utils/config-params';
+import Paper from '@mui/material/Paper';
+
+export const styles = {
+    illustration: {
+        backgroundColor: 'transparent',
+        backgroundImage: 'none',
+        boxShadow: '0',
+        padding: '0px 0px',
+    },
+};
+
+/**
+ * This component displays a hide able explanation illustration
+ */
+const LineSplitWithVoltageLevelIllustration = () => {
+    const [showDiagram, setShowDiagram] = useState(true);
+    const intl = useIntl();
+    const theme = useSelector((state: ReduxState) => state[PARAM_THEME]);
+
+    useEffect(() => {
+        // dynamically modify the text inside the svg :
+        const replacedTexts = [
+            {
+                eltId: 'voltageLevelOrigBeforeTxt',
+                tradId: 'VoltageLevel1',
+            },
+            {
+                eltId: 'voltageLevelExtrBeforeTxt',
+                tradId: 'VoltageLevel2',
+            },
+            {
+                eltId: 'voltageLevelOrigAfterTxt',
+                tradId: 'VoltageLevel1',
+            },
+            {
+                eltId: 'voltageLevelToSplitAtAfterTxt',
+                tradId: 'VoltageLevelToSplitAt',
+            },
+            {
+                eltId: 'voltageLevelExtrAfterTxt',
+                tradId: 'VoltageLevel2',
+            },
+            {
+                eltId: 'lineToSplitBeforeTxt',
+                tradId: 'LineToSplit',
+            },
+            {
+                eltId: 'line1AfterTxt',
+                tradId: 'Line1',
+            },
+            {
+                eltId: 'line2AfterTxt',
+                tradId: 'Line2',
+            },
+        ];
+
+        replacedTexts.forEach((replacedText) => {
+            const elt = document.getElementById(replacedText.eltId);
+            if (elt != null) {
```

Comment:
type coercion in JavaScript can sometimes lead to unexpected results, so it's mostly recommended to use the strict equality operator .. 

```suggestion
            if (elt !== null) {
```
or you can shortly use 
```suggestion
            if (!!elt) {
```

Reply:
Done.